NGitLab.Mock/Clients/CommitClient.cs

```diff
@@ -38,5 +42,20 @@ public JobStatus GetJobStatus(string branchName)
         {
             throw new NotImplementedException();
         }
+
+        public GitLabCollectionResponse<Models.MergeRequest> GetRelatedMergeRequestsAsync(RelatedMergeRequestsQuery query)
+        {
+            using (Context.BeginOperationScope())
+            {
+                var project = GetProject(_projectId, ProjectPermission.View);
+
+                var mergeRequests = project.MergeRequests.Where(mr => mr.Commits.SingleOrDefault(
+                        commit => commit.Sha.Equals(query.Sha.ToString(), StringComparison.OrdinalIgnoreCase)) != null);
+
+                var relatedMerqueRequests = mergeRequests?.Select(mr => mr.ToMergeRequestClient());
```

Comment:
`mergeRequests?` => is the nullable check necessary?

Reply:
Nope, I fixed that :)